conf/evolutions/collins/14.sql

```diff
@@ -0,0 +1,11 @@
+# --- Add GPU information to asset_meta
+
+# --- !Ups
+
+INSERT INTO asset_meta (name, priority, label, description) VALUES ('GPU_PRODUCT', -1, 'GPU Product', 'GPU product (description of GPU)');
```

Comment:
doesnt this need a GPU_DESCRIPTION as well? I see we extract description from LSHW, but it doesnt seem like it ever gets persisted